bedrock/exp/urls.py

```diff
@@ -19,4 +19,5 @@
     page('firefox/mobile', 'exp/firefox/mobile.html', ftl_files=['firefox/mobile']),
     url(r'^$', views.home_view, name='exp.mozorg.home'),
     page('firefox/accounts', 'exp/firefox/accounts.html', ftl_files=['firefox/accounts']),
+    page('firefox/unfck', 'exp/firefox/campaign/unfck/index.html', active_locales=['en-US', 'en-GB', 'en-CA']),
```

Comment:
The main production page is only enabled for `en-US` - shouldn't both pages match?